### PR TITLE
#6 Save users in database with default role guest

### PR DESCRIPTION
**Pull request by reqiii**

### CODE REVIEW

File: src/main/kotlin/org/arctgkolbasy/bot/user/model/Roles.kt [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package org.arctgkolbasy.bot.user.model
+
+import jakarta.persistence.*
+import org.arctgkolbasy.bot.user.UserRoles
+
+@Entity
+@Table(name = "roles")
+class Roles (
```
Comment (DimaZet) [inline src/main/kotlin/org/arctgkolbasy/bot/user/model/Roles.kt:8]: rename class name ```Roles``` to ```Role``` please

File: src/main/kotlin/org/arctgkolbasy/bot/user/User.kt [hunk 2]
```diff
@@ -1,11 +1,11 @@
 package org.arctgkolbasy.bot.user
 
-data class User(
+data class User (
```
Comment (deradot22) [inline src/main/kotlin/org/arctgkolbasy/bot/user/User.kt:3]: odd space

File: src/main/resources/db/migration/changesets/1-create-tables.sql [hunk 3]
```diff
@@ -1,9 +1,10 @@
...
+  telegram_id bigint unique not null,
   username text unique not null,
-  firstname text,
+  firstname text not null,
   lastname text,
-  user_role text not null
+  is_bot boolean not null
 );
 
 create table if not exists products (
```
Comment (DimaZet) [inline src/main/resources/db/migration/changesets/1-create-tables.sql:10]: rename please tables 'consumer' and 'users_to_role' to 'consumers' and 'users_to_roles'

**Review by DimaZet — CHANGES_REQUESTED** [review]
Looks good to me but few naming fixes needed

**Review by DimaZet — APPROVED** [review]

**Pull request merged**